core/signeddata.go

```diff
@@ -637,6 +638,77 @@ func (r *VersionedSignedValidatorRegistration) UnmarshalJSON(input []byte) error
 	return nil
 }
 
+// NewPartialSignedRandao is a convenience function that returns a new partially signed Randao Reveal.
+func NewPartialSignedRandao(randao eth2util.SignedEpoch, shareIdx int) ParSignedData {
```

Comment:
probably fine to take epoch, sig, index directly like previously since it seems like we never have a `eth2util.SignedEpoch` available.